local/blockchain.go

```diff
@@ -777,12 +777,18 @@ func (ln *localNetwork) createBlockchainConfigFiles(
 		// update config info for snapshopt/restart purposes
 		// put into defaults and reset node specifics
 		if chainSpec.ChainConfig != nil {
+			if ln.chainConfigFiles == nil {
```

Comment:
the problem is a nil when creating the network, I would rather init the maps there, to avoid any posible conflict afterwards (the issue belongs to loadSnapshot)